Makefile

```diff
@@ -17,6 +17,5 @@ clean:
 # find the most recent working version and leave a copy in the file
 # called corepop at top-level.
 corepop:
-	COREPOP=`/bin/sh ./find.sh 2>/dev/null` && [ ! -z "$$COREPOP" ] && cp -p "$$COREPOP" corepop
+	COREPOP=`./find.sh` && [ ! -z "$$COREPOP" ] && cp -p "$$COREPOP" corepop
```

Comment:
Why allow the misleading error output to be visible?

Reply:
Because otherwise we flush away our error message that this commit adds.

Reply:
The `find.sh` script runs the corepops with 
```
`./$corepop ":sysexit()" 2>&1`
```
So the corepop output will be entirely captured in the command substitution.